src/app/__tests__/App.test.tsx

```diff
@@ -145,7 +145,7 @@ describe('src/app/App', () => {
             await jest.advanceTimersToNextTimer();
         });
 
-        expect(screen.getByText(/You do not have access to Notifications/i)).toBeTruthy();
+        expect(screen.getByTestId('content')).toBeTruthy();
```

Comment:
Almost there!

Before this change, we were testing the read permissions before the routes. This is no longer true, as the check is inside the routes now.

To fix this, we could mock the [ConnectedIntegrationsListPage](https://github.com/RedHatInsights/notifications-frontend/blob/master/src/Routes.tsx#L33) and [NotificationsListPage](https://github.com/RedHatInsights/notifications-frontend/blob/master/src/Routes.tsx#L41) components.

We no longer need to mock the [routes](https://github.com/RedHatInsights/notifications-frontend/blob/da8e0a816af631eaf147b3a22f31e1ba7c500fea/src/app/__tests__/App.test.tsx#L21-L26)

```
jest.mock('../../Routes', () => { // <--- this is the import path to what you want to mock
    const MockedRoutes: React.FunctionComponent = () => <div data-testid="content" />; // something to mock
    return {
        Routes: MockedRoutes // "Routes" is what is being exported, in the case of ConnectedIntegrationsListPage it would be `ConnectedIntegrationsListPage` because that's the name of what we export 
    };
});
```

So we would need a similar block as above for [ConnectedIntegrationsListPage](https://github.com/RedHatInsights/notifications-frontend/blob/master/src/Routes.tsx#L33) and [NotificationsListPage](https://github.com/RedHatInsights/notifications-frontend/blob/master/src/Routes.tsx#L41).

Reply:
Those components are already mocked in that file..

`jest.mock('../pages/Notifications/List/Page', () => ({
    NotificationsListPage: () => 'Notifications'
}));

jest.mock('../pages/Integrations/List/Page', () => ({
    ConnectedIntegrationsListPage: () => 'Integrations'
}));`

Would the above be to replace that or in addition to?

Reply:
Would replace the Routes, also you can wrap the 'Notifications' and 'Integrations' text in a: `<div data-testid="content" />` to prevent from affecting other tests